docs/source/guide.rst

```diff
@@ -220,17 +174,49 @@ predicate function via the ``retry_when`` parameter::
   def count_words(url):
     ...
 
+If you want to use a different strategy than the default exponential backoff to define how long to wait between retries, you can pass the `backoff_strategy` keyword argument to |actor|. For instance to retry every minute::
+
+   @remoulade.actor(min_backoff=60000, backoff_strategy='constant')
+   def count_words(URL):
+     ...
+
 The following retry options are configurable on a per-actor basis:
 
-===============  ============  =====================================================================================================================
-Option           Default       Description
-===============  ============  =====================================================================================================================
-``max_retries``  ``0``         The maximum number of times a message should be retried.  ``None`` means the message should be retried indefinitely.
-``min_backoff``  15 seconds    The minimum number of milliseconds of backoff to apply between retries.  Must be greater than 100 milliseconds.
-``max_backoff``  7 days        The maximum number of milliseconds of backoff to apply between retries.  Higher values are less reliable.
-``retry_when``   ``None``      A callable that determines whether or not a message should be retried.  When this is set, ``max_retries`` is ignored.
-===============  ============  =====================================================================================================================
+max_retries
+^^^^^^^^^^^
+
+The maximum number of times a message should be retried. Default to ``0``.
+
+min_backoff
+^^^^^^^^^^^
+
+The minimum number of milliseconds of backoff to apply between retries.  Must be greater than 100 milliseconds. Defaults to 15 seconds.
+
+max_backoff
+^^^^^^^^^^^
+
+The maximum number of milliseconds of backoff to apply between retries. Higher values are less reliable. Defaults to 1 hour.
+
+retry_when
+^^^^^^^^^^
+
+A callable that determines whether or not a message should be retried.  When this is set, ``max_retries`` is ignored. Defaults to ``None``.
```

Comment:
Takes the number of retries so far and the exception as input, and expect a boolean as output.